ogb/utils/mol.py

```diff
@@ -7,7 +7,7 @@
 def ReorderAtoms(mol):
```

Comment:
Change the function name to reflect "how" you reorder the atom.

Reply:
Have changed the function name to be `CanonicalRankAtoms` in consistant to rdkit.

Reply:
Maybe we could convert SMILES and .sdf to graphs in `PygPCQM4Mv2Dataset` simultaneously.

Reply:
How about `ReorderCanonicalRankAtoms`?

And let's make sure backward compatibility not to disturb anyone who already calls this function.

Reply:
Sure. Have changed the function name and set the `reorder_atoms` as an option.

Reply:
thanks! LGTM.

Reply:
Would give a unified `PygPCQM4Mv2Dataset` later.

Reply:
Sounds good! Thanks. For the next PR, it would be nice for you to contribute to [example](https://github.com/snap-stanford/ogb/tree/master/examples/lsc/pcqm4m-v2). This way, we know and can test how to use these features.